src/MSAAdvanced.sol

```diff
@@ -254,7 +254,7 @@ contract MSAAdvanced is
                 if (signer != address(this)) {
                     return VALIDATION_FAILED;
                 }
-
+                _addStorageBase(MODULEMANAGER_STORAGE_LOCATION);
```

Comment:
this feels unecessary since if you just use the eoa as owner without modules then this storage will not actually be used and if you do init the account then you pay for this twice